module/VuFind/src/VuFind/Controller/CombinedController.php

```diff
@@ -169,8 +165,6 @@ public function resultsAction()
             ->get(\VuFind\Search\Options\PluginManager::class);
         $config = $this->serviceLocator->get(\VuFind\Config\PluginManager::class)
             ->get('combined')->toArray();
-        $supportsCart = false;
```

Comment:
I confess that I haven't tested to confirm, but I'm concerned that this bit might be broken by the refactoring. The combined search displays the top-level cart/bulk controls based on whether ANY of the columns support the cart. Unless this logic/check is built into the Combined Options class, I think it's getting lost. Did you check this already? If not, let me know if you need help testing.

Reply:
You were right, Combined Options didn't return correct information. Now fixed.